modules/@apostrophecms/schema/ui/apos/components/AposInputWrapper.vue

```diff
@@ -124,43 +148,5 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-  .apos-field {
```

Comment:
Why move these. Are they used by another? All inputs are wrapped by this.

Reply:
moved back